src/components/HorizontalTab/index.tsx

```diff
@@ -17,25 +17,25 @@ const HorizontalTab = ({selectedTab, onSelectedTabChange, tabs = []}: Horizontal
   const styles = StyleSheet.create({
     tabs: {
       width,
-      borderBottomColor: COLORS.lightgrey,
+      borderBottomColor: COLORS.gray100,
       borderBottomWidth: 1,
       flexDirection: 'row',
       alignItems: 'center',
       justifyContent: 'center',
-      backgroundColor: COLORS.white,
+      backgroundColor: COLORS.almostBlack,
       display: 'flex'
     },
     tabsFixed: {
       width,
-      borderBottomColor: COLORS.lightgrey,
+      borderBottomColor: COLORS.gray100,
       borderBottomWidth: 1,
       paddingLeft: 20,
       paddingRight: 20,
       flexDirection: 'row',
       position: 'absolute',
       top: 42,
       zIndex: 2000,
-      backgroundColor: COLORS.white
+      backgroundColor: COLORS.almostBlack
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the color changes here also need to be checked for consistency with the overall theme and user experience. 

```diff
-      borderBottomColor: COLORS.lightgrey,
+      borderBottomColor: COLORS.gray100,
       borderBottomWidth: 1,
       paddingLeft: 20,
       paddingRight: 20,
       flexDirection: 'row',
       position: 'absolute',
       top: 42,
       zIndex: 2000,
-      backgroundColor: COLORS.white
+      backgroundColor: COLORS.almostBlack
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->